### ISSUE TITLE
No Access above my Home Directory possible

### ISSUE THREAD

**Issue by MalteMart** [microsoft/vscode-remote-release#4150]
Hello,

I'm currently working on my master thesis and need to access my code via SSH on the Uni computers. My programs are located on the Uni cluster. If I use a terminal on Linux I get to my home directory via ssh and can navigate out of it to my directory on the cluster. 
Now if I want to do the same in VS Code using remote SSH, I can access my home directory and everything in it , but I cannot navigate out of the "home" directory in which my home dir is. So when I click "open folder" and click the ".." option to move up a dir, nothing happens as soon as I reach the "home" dir. I think usually only the home directory is accessible, if you weren't granted access to the cluster; maybe that has something to do with the problem.

I have also tried it while being connected to the Uni Network using VPN, but couldn't get a connection, as I have no connection to the Internet then. If I check the "Use this connection only for ressources on this network" option; I have internet connection again, but face the same problem as before.

Can anyone help me with this or tell me if there is a way to solve this?  I'm especially confused about that I can access the folder in a terminal (even the Terminal in VS Code), but not open it in VS Code. 

Thanks

I'm working on VS Code 1.51.1 on Ubuntu 20.04. (though the same problem occurs on Windows 10) and using the Remote development Extension with Remote - SSH 0.56.0. 
My Extensions are: 
Remote - Containers 0.148.1 
Remote - SSH 0.56.0
Remote - SSH: Editing Configuration Files 0.58.0
Remote - WSL 0.51.4
Remote Development 0.19.0

**Comment by roblourens** [comment]
We would usually say "above" your home directory, it's fine, just to avoid confusion if someone else looks at this issue. What happens if you open a file/folder outside of the home directory through the integrated terminal? Like `code somefile.txt`

**Comment by MalteMart** [comment]
@roblourens Ok, thank you, I edited it to "above" now. So there have been some developments regarding this problem:

After looking into it more, I assume that the actual problem is not restricted access, but simply that there are a lot of directories in the "users" directory. What looked like VS Code not allowing me to navigate to the above dir, might actually just be the integrated Filebrowser (or remote ssh) crashing due to the many directories. After this problem, connecting to the host again took about 10 minutes, but after that it works fine again. I just have to access the folder via the Terminal (without trying to access it first via the "Open Folder" option which still causes the connection to break and reconnecting to take a lot of time). 
I think disabling Remote.SSH: Use Local Server might have solved an additional timeout problem. 

Oddly at first I was able to open a file, but not the folder containing this file. It seems to be working now though (fingers crossed).

Thank you!
--------------------------------------------------------------------------------------------------------------------------------------
Here is the error I received when trying to open the folder before changing the Local Server option: Could not establish connection to "HOSTNAME.de": The VS Code Server failed to start (full error message attached)
[Fehlermeldung.txt](https://github.com/microsoft/vscode-remote-release/files/5656243/Fehlermeldung.txt)
